warehouse/query-core/src/test/java/datawave/query/config/ShardQueryConfigurationTest.java

```diff
@@ -434,17 +436,46 @@ public void testIsTldQuery() {
      */
     @Test
     public void testCheckForNewAdditions() throws IOException {
-        int expectedObjectCount = 172;
+        int expectedObjectCount = 173;
         ShardQueryConfiguration config = ShardQueryConfiguration.create();
         ObjectMapper mapper = new ObjectMapper();
         JsonNode root = mapper.readTree(mapper.writeValueAsString(config));
         Iterator<JsonNode> rootIter = root.elements();
         int objectCount = 0;
         while (rootIter.hasNext()) {
-            rootIter.next();
+            JsonNode next = rootIter.next();
+            
             objectCount++;
         }
         
-        Assert.assertEquals("New variable was added to or removed from the ShardQueryConfiguration", expectedObjectCount, objectCount);
+        assertEquals("New variable was added to or removed from the ShardQueryConfiguration", expectedObjectCount, objectCount);
+    }
+    
+    @Test
+    public void whenRetrievingActiveQueryLogName_givenTableNameSource_thenReturnsTableName() {
```

Comment:
ouch!  How about a shorter name and some comments above it.

Reply:
I agree that one shouldn't use overly long method names when writing product code, but I don't think the same should apply to unit tests. By using a when<behavior_under_test>_given<factor>_then<expected_behavior>, I'm able to state a single technical requirement being tested (which works well for TDD) and make the method self documenting. If a test fails during a build, I'll be able to know at a glance in the logs what was being tested and which requirement failed. I think this naming style also makes it easier for other developers to clearly understand each test case.

I have found that when testing multiple methods with multiple technical requirements for each method, writing short but still clear method names can get pretty tricky. Granted, that's not necessarily the case here and I can shorten the names a bit. I hope I've been able to explain my reasoning clearly; this is a habit formed from many years of TDD.